pkg/loader/executor.go

```diff
@@ -17,6 +17,7 @@ import (
 	"context"
 	gosql "database/sql"
 	"fmt"
+	"github.com/godror/godror"
```

Comment:
We should add some unit tests for oracle part in this file. Especially for the safe mode part.